app/modules/ads/index.js

```diff
@@ -387,6 +388,11 @@ class Ads {
   }
 
   waitForUapResponse(uapCallback, noUapCallback) {
```

Comment:
`waitForUapResponse` is used outside ads "namespace" (in the `smart-banner.js`). Because of that we need to handle not resolved callbacks when there is no ad-engine (e.g. in fastboot run)